### PR TITLE
Solucion problema 876

### PR DESCRIPTION
**Pull request by Eduardo-93**

### CODE REVIEW

File: algorithm_exercises/876. Middle of the Linked List/solution.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Definition for singly-linked list.
```
Comment (alnavarroh) [inline algorithm_exercises/876. Middle of the Linked List/solution.py]: Remove this comments pls

**Review by alnavarroh — CHANGES_REQUESTED** [review]

File: algorithm_exercises/876. Middle of the Linked List/solution.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+class Solution:
+    def middleNode(self, head: Optional[ListNode]) -> Optional[ListNode]:
+        curr = curr2 = head
+        while curr2 and curr2.next:
+            curr = curr.next
+            curr2 = curr2.next.next
+        return curr
```
Comment (Darova93) [inline algorithm_exercises/876. Middle of the Linked List/solution.py]: Add an empty line at the end of the file.
Also we don't need the Solution class

**Review by Darova93 — CHANGES_REQUESTED** [review]

**Comment by Bydel93** [comment]
lgtm

**Review by Bydel93 — APPROVED** [review]

**Review by danielalbinoms — APPROVED** [review]
thumbs up

File: algorithm_exercises/876. Middle of the Linked List/solution.py (hunk 2 above)
Comment (Eduardo-93) [reply algorithm_exercises/876. Middle of the Linked List/solution.py]: Done

File: algorithm_exercises/876. Middle of the Linked List/solution.py (hunk 1 above)
Comment (Eduardo-93) [reply algorithm_exercises/876. Middle of the Linked List/solution.py]: 👍

**Review by alnavarroh — APPROVED** [review]

**Review by alnavarroh — CHANGES_REQUESTED** [review]
please remove pycache directory kubo

**Review by danielalbinoms — APPROVED** [review]

**Pull request closed without merging**